### DISCUSSION TITLE
Problem sidebar with Inertia

### DISCUSSION THREAD

**Discussion by diogofm7** [mdbootstrap/TW-Elements#1908] [Support from community]
I have a problem when using tailwind-elements and Laravel Inertia, when changing pages using <Link>, where the page is not fully reloaded, the sidebar closes, but the crawler is not removed, staying on top of the content .

**Comment by juujisai** [comment]
Hi! Did you use our laravel tutorial? 

https://tailwind-elements.com/docs/standard/integrations/laravel-integration/
